### ISSUE TITLE
Error/Warning Message for an Unrecognized XML Tag

### ISSUE THREAD

**Issue by latessa** [JETSCAPE/JETSCAPE#132]
It was suggested to output an error or warning message if the user xml file contains an unrecognized tag.  Currently an unrecognized tag is simply ignored.

**Comment by latessa** [comment]
@LSchwiebert I looked through the example XML files and as far as I saw, there are no legitimate user XML tags that aren't also included in the main XML file.  I don't think there's a list anywhere of the known or recognized XML tags other than what's provided in the main XML file.  So, I've implemented this to build a searchable list of tag names from the main XML file and issue a warning if a tag found in the user XML file isn't included in the list.  I included this as a warning instead of an error just in case there's ever a situation where a legitimate user tag for some reason isn't included in the main XML file, but I haven't encountered such a scenario.

Here's a [link ](https://github.com/JETSCAPE/JETSCAPE/commit/006e54cb9d6eab6065e45cf1469750688a5c87a4) to my branch and commit showing the changes.  Since you're familiar with this section of code having worked on updating the XML filenames, does this look like an OK way to implement this functionality?
